project/urls.py

```diff
@@ -17,9 +17,16 @@
     url(r'^$', APIRootView.as_view(), name=APIRootView.URL_NAME),
 ]
 
+urlpatterns += [
+    url(r'^forms/(?P<pk>[0-9]+)/$', CreateForm.as_view(), name=CreateForm.URL_NAME),
+]
 
 if settings.DEBUG:
     urlpatterns += [
         url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
     ]
+
+
+
+
```

Comment:
@szymonzmyslony There are too many empty lines here.
This should have been picked up by the commit hooks.